### PR TITLE
Demo mobitru integration

### PR DESCRIPTION
**Pull request by DPrakapuk**
added additional folder for integration with mobitru service

### CODE REVIEW

File: mobile-web/mobitru.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            rpConfig: {
+                token: RP_TOKEN,
+                endpoint: RP_ENDPOINT,
+                description: 'Description',
+                tags: ['Tag'],
+                project: RP_PROJECT,
+                launch: RP_LAUNCH
+            },
+        },
+        service: [appiumConfig],
```
Comment (AlexGalichenko) [inline mobile-web/mobitru.js:58]: appiumConfig is not needed here

File: mobile-web/mobitru.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const encodeKey = encodeURIComponent(API_KEY);
+const PROJECT_NAME = process.env.PROJECT_NAME;
+const PLATFORM_NAME = process.env.PLATFORM_NAME;
+const BROWSER_NAME = process.env.BROWSER_NAME;
+const DEVICE_NAME = process.env.DEVICE_NAME;
+const RP_TOKEN = process.env.RP_TOKEN;
+const RP_ENDPOINT = process.env.RP_ENDPOINT;
+const RP_PROJECT = process.env.RP_PROJECT;
+const RP_LAUNCH = process.env.RP_LAUNCH;
+const appiumConfig = wdioService([
```
Comment (AlexGalichenko) [inline mobile-web/mobitru.js:15]: appium config is not needed here

**Review by AlexGalichenko — CHANGES_REQUESTED** [review]

File: mobile-web/mobitru.js (hunk 2 above)
Comment (nomid0612) [reply mobile-web/mobitru.js]: removed

File: mobile-web/mobitru.js (hunk 1 above)
Comment (nomid0612) [reply mobile-web/mobitru.js]: removed

**Review by KirillBogdanets — APPROVED** [review]

**Review by AlexGalichenko — APPROVED** [review]

**Pull request merged**